plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -15,7 +15,6 @@
 
```

Comment:
No properties name change here? No docs changes? No tests changes?

Reply:
Not at this moment for this module, but it will likely change. In fact, whenever I add additional properties to `AwsCredentialsProviderConfig`, they're automatically available in all the modules that are using it, so they should be documented in them (if there's documentation at all, that is).